daemon/cluster/executor/container/container.go

```diff
@@ -656,11 +657,11 @@ func networkCreateRequest(name string, nw *api.Network) clustertypes.NetworkCrea
 			Options: nw.IPAM.Driver.Options,
 		}
 		for _, ic := range nw.IPAM.Configs {
-			req.IPAM.Config = append(req.IPAM.Config, network.IPAMConfig{
-				Subnet:  ic.Subnet,
-				IPRange: ic.Range,
-				Gateway: ic.Gateway,
-			})
+			var cfg network.IPAMConfig
+			cfg.Subnet, _ = netip.ParsePrefix(ic.Subnet)
+			cfg.IPRange, _ = netip.ParsePrefix(ic.Range)
+			cfg.Gateway, _ = netip.ParseAddr(ic.Gateway)
+			req.IPAM.Config = append(req.IPAM.Config, cfg)
```

Comment:
Same consideration here as my other comment in the other packages; perhaps a utility would make it more clear that this is all intentional and can act as good location to describe that.